pages/about.html

```diff
@@ -0,0 +1,113 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Super SportCar Global Race | About</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="/Assets/sass/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="scroll bg-light" id="body">
+  <div class="container-fluid">
+    <header class="row">
+      <div class="social">
+        <ul>
+          <li class="social-icon"><a href="#">Facebook <i class="fab fa-facebook"></i></a></li>
+          <li class="social-icon"><a href="#">Instagram <i class="fab fa-instagram"></i></a></li>
+          <li class="social-icon"><a href="#">Twitter <i class="fab fa-twitter"></i></a></li>
+        </ul>
+      </div>
+      <nav class="navbar navbar-expand-lg navbar-light">
+          <a class="navbar-brand" href="/index.html"><i class="fas fa-car"></i></a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="../index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+            </ul>
+          </div>
+      </nav>
+      <div class="col-12">
+        <div class="row heading">
+          <h1 class="col-12 col-md-8 order-2">Super Sportcar Global Race 2021</h1>
+          <h2 class="col-12 order-1">"Hello sharing World"</h2>
+        </div>
+        <div class="col-12 header-info">
+          <div class="row header-info-container">
+            <p class="col col-md-8">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Expedita tempora pariatur provident minima.
+            </p>
+            <h4 class="col-11">Please Contact Us Lorem ipsum dolor, sit amet consectetur adipisicing elit. Obcaecati molestias vero corporis deleniti</h4>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main class="row main">
+      <section class="col main-program">
+        <div class="program-heading">
+          <h2>Super SportCars Ficticional Global Race 2021</h2>
+          <span></span>
+          <p>The loogo of the ficticional race Lorem ipsum dolor sit amet consectetur adipisicing elit. consequuntur</p>
+        </div>
+        <div class="race-logo">
+            <img src="/Assets/img/race-car.png" alt="">
```

Comment:
- [ ] **Please add relevant `alt` text to your images.**

Adding `alt` text improved the accessibility on your page and benefits the overall ranking. 

***Please make sure that every other image contains a relevant alt text**